src/index.ts

```diff
@@ -42,16 +42,21 @@ export const validator = async (token: string, accountSid: string, authToken: st
       reject('Unauthorized: AccountSid or AuthToken was not provided');
       return;
     }
+    
+    // detects if authToken is an object with attributes key and secret to assign hasAPIKey a boolean 
+    const hasAPIKey = !!apikeys ? (!!apikeys.key ? (!!apikeys.secret ? true : false ) : false) : false;
+
+    // changed to include API keys if hasAPIKey is true
+    const authorization = hasAPIKeys ? Buffer.from(`${apikeys.key}:${apikeys.secret}` : Buffer.from(`${accountSid}:${authToken}`);
 
-    const authorization = Buffer.from(`${accountSid}:${authToken}`);
     const requestData = JSON.stringify({ token });
     const requestOption = {
       hostname: 'iam.twilio.com',
       port: 443,
       path: `/v1/Accounts/${accountSid}/Tokens/validate`,
       method: 'POST',
       headers: {
-        Authorization: `Basic ${authorization.toString('base64')}`,
+        'Authorization': `Basic ${authorization.toString('base64')}`,
```

Comment:
Remove quotation please. I should also add a proper linter to this repo!

Reply:
Quotes have been removed, though I would argue it's inconsistent with the other item attributes,
eg
```
headers: {
        Authorization: `Basic ${authorization.toString('base64')}`,
        'Cache-Control': 'no-cache',
        'Content-Type': 'application/json',
        'Content-Length': requestData.length,
      },
```